### PR TITLE
Add Web Security

### PR DESCRIPTION
**Pull request by qazbnm456**
This PR adds **Awesome Web Security** list.

**[https://github.com/qazbnm456/awesome-web-security](https://github.com/qazbnm456/awesome-web-security)**

**A curated list of Web Security materials and resources.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by jbmoelker** [comment]
In the license paragraph: I think it should be you who waives all rights, not Sindre.

Just something I noticed. I'm not a maintainer.

**Comment by qazbnm456** [comment]
@jbmoelker You're right, and I've made the change! Thanks for your suggestion. 😄

**Comment by davisonio** [comment]
A lot of these resources don't have descriptions - e.g. why should I follow the users listed in the Twitter section?

**Comment by qazbnm456** [comment]
@davisonio Thanks for your reminder. I think I've added descriptions for those entries.

File: readme.md [hunk 1]
```diff
@@ -380,6 +380,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Honeypots](https://github.com/paralax/awesome-honeypots) - Deception trap, designed to entice an attacker into attempting to compromise the information systems in an organization.
 - [Incident Response](https://github.com/meirwah/awesome-incident-response)
 - [Vehicle Security and Car Hacking](https://github.com/jaredmichaelsmith/awesome-vehicle-security)
+- [Web Security](https://github.com/qazbnm456/awesome-web-security) - Security Resources for Web Security.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should include a short description about the list project/theme **(not the list itself)**

**Comment by sindresorhus** [comment]
> has a Table of Contents section named Contents as the first section,

Not followed.

> includes a succinct description of the project/theme at the top (example),

Not followed.

> Drop all the A / An prefixes in the descriptions.

Not followed.

---

The repo should have the following GitHub topics set: awesome-list, awesome, list. I encourage you to add more relevant topics.

---

`Collection` and `Resource` are not very good section titles.

---

`Url` => `URL`

`Dns` => `DNS`

---

I'd like to see some links to community resources: Stack Overflow tag, IRC, chat, forums, communities, Reddit, etc. Example: https://github.com/sindresorhus/awesome-electron#community

---

Section titles should be pluralized. `Blog` => `Blogs`

`Miscellaneous` should be the last section.

**Comment by qazbnm456** [comment]
@sindresorhus Thank you for suggestions. I've made all the modifications.

**Comment by sindresorhus** [comment]
Please read https://github.com/sindresorhus/awesome/pull/898#discussion_r112403796 again. It's not correct. The description should not be about the list itself.

**Comment by qazbnm456** [comment]
@sindresorhus How about `Tips for penetrating Web`? Thank you.

**Comment by sindresorhus** [comment]
@qazbnm456 That is still about the list and not the topic itself.


---

`Useful Resources` => `Resources`. All the resources should be useful, so no need to be so verbose.

---

All your links like `Link, written by PHITHON.` should be `Link - Written by PHITHON.`

**Comment by qazbnm456** [comment]
@sindresorhus I think I've misunderstood your word before, how about this now? Should be more clear. Thank you.

**Pull request merged**